packaging/msi/Product.wxs

```diff
@@ -0,0 +1,111 @@
+
+            <!-- The IceGrid GUI jar -->
+            <File Source="!(bindpath.SOURCE_ROOT)\java\lib\icegridgui.jar" Directory="BinFolder" />
+
+            <!-- Slice Files-->
+            <Files Include="!(bindpath.SOURCE_ROOT)\slice\**" Directory="SliceFolder"/>
+
+            <!-- Config Files -->
+            <Files Include="!(bindpath.SOURCE_ROOT)\cpp\config\*.cfg" Directory="ConfigFolder"/>
+            <Files Include="!(bindpath.SOURCE_ROOT)\cpp\config\templates.xml" Directory="ConfigFolder"/>
+
+            <!-- License Files -->
+            <File Id="License" Source="!(bindpath.SOURCE_ROOT)\packaging\msi\docs\LICENSE.txt" />
+            <File Id="IceLicense" Source="!(bindpath.SOURCE_ROOT)\packaging\msi\docs\ICE_LICENSE.txt" />
+            <File Id="ThirdPartyLicense" Source="!(bindpath.SOURCE_ROOT)\packaging\msi\docs\THIRD_PARTY_LICENSE.txt" />
+        </ComponentGroup>
+
+        <!-- Define the components that install the start menu shortcuts -->
+        <ComponentGroup Id="StartMenuComponents">
+            <Component
+                Id="AppStartMenuShortcuts"
+                Directory="MyAppStartMenu"
+                Guid="618E2B68-1829-478A-AB46-47D65887F351">
+                <util:InternetShortcut
+                    Id="ManualLink"
+                    Name="Ice Manual"
+                    Type="url"
+                    Target="https://doc.zeroc.com/ice/3.8/" />
+                <util:InternetShortcut
+                    Id="SampleProgramsLink"
+                    Name="Ice Sample Programs"
+                    Type="url"
+                    Target="https://github.com/zeroc-ice/ice-demos/tree/v3.8.0" />
+                <util:InternetShortcut
+                    Id="ReleaseNotesLink"
+                    Name="Release Notes"
+                    Type="url"
+                    Target="https://doc.zeroc.com/rel/ice-releases/ice-3-8/ice-3-8-0-release-notes" />
+                <util:InternetShortcut
+                    Id="SourceCodeLink"
+                    Name="Source Code on Github"
+                    Type="url"
+                    Target="https://github.com/zeroc-ice/ice/tree/v3.8.0" />
+                <util:InternetShortcut
+                    Id="UsingBinDistLink"
+                    Name="Using this Binary Distribution"
+                    Type="url"
+                    Target="https://doc.zeroc.com/rel/ice-releases/ice-3-8/ice-3-8-0-release-notes/using-the-windows-binary-distributions-for-ice-3-8-0" />
+                <Shortcut
+                    Id="IceGridGUIShortcut"
+                    Name="IceGrid GUI"
+                        Target="[BinFolder]icegridgui.jar"
+                        WorkingDirectory="BinFolder"
+                        Icon="IceGridGUIIcon"
+                        Advertise="no"/>
+                <RegistryValue Root="HKCU" Key="ZeroC\Ice-3.8.0\Installed" Value="1." Type="string" KeyPath="yes" />
+                <RemoveFolder Id="MyAppStartMenu" On="uninstall"/>
```

Comment:
Is this Id correct?

Reply:
Can we change this `MyAppStartMenu` name?

Reply:
What about `IceAppStartMenu` or do you have a better suggestion?